size-stats.json

```diff
@@ -1,15 +1,15 @@
 {
     "dist": {
-        "js": "6898.15 KB",
-        "jsNoMisticaIcons": "797.35 KB"
+        "js": "6898.29 KB",
+        "jsNoMisticaIcons": "797.49 KB"
     },
     "distEs": {
-        "js": "5536.45 KB",
+        "js": "5536.44 KB",
         "jsNoMisticaIcons": "664.20 KB"
     },
     "libOverhead": {
         "initial": "127.79 KB",
-        "withMistica": "267.93 KB",
-        "difference": "140.14 KB"
+        "withMistica": "273.31 KB",
+        "difference": "145.52 KB"
```

Comment:
I think this 5KB increase is due to the change in `package-version.tsx`
The CRA we use to measure this lib overhead is using webpack 4 and it seems it can not tree-shake the package.json

Reply:
we could make a script that runs before build and writes the version into a file.